libs/loggers/source_loggers/file_system_logger.py

```diff
@@ -32,7 +32,8 @@ def save_screenshot(self, cv_image):
 
     def log_objects(self, objects, violating_objects, violating_objects_index_list, violating_objects_count,
                     detected_objects_cout, environment_score, time_stamp, version):
-        file_name = str(date.today())
+        
+        file_name = time_stamp[0: 10]
```

Comment:
Is this always safe to do?

Reply:
Yes, the time_stamp parameter always receives the result of apply `strftime("%Y-%m-%d %H:%M:%S")` to a datetime.